install_config/registry/accessing_registry.adoc

```diff
@@ -272,3 +272,95 @@ cf2616975b4a: Image successfully pushed
 Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
+
+[[access-reading-and-writing-image-signatures]]
+=== Reading and Writing Image Signatures
+The {product-title} integrated Docker Registry provides an "extensions" endpoint that
+allows you to write and read image signatures. The image signatures are stored in
+{product-title} API, but this endpoint allows you to read and write them without
+having direct access to the {product-title} API endpoint.
+
+[NOTE]
+====
+This endpoint is experimental and not supported by the upstream Docker Registry project.
+See https://docs.docker.com/registry/spec/api/[upstream API documentation] for general
+information about the docker registry API.
+====
+
+==== Writing Image Signatures
+In order to add a new signature to the Image, users can use the HTTP `PUT` method together
+with the JSON payload sent the extensions endpoint:
+
+====
+----
+PUT /extensions/v2/{namespace}/{name}/signatures/sha256:{digest}
+----
+====
+
+====
+----
+$ curl -X PUT --data @signature.json http://<user>:<token>@<registry-endpoint>:5000/extensions/v2/<namespace>/<name>/signatures/sha256:<digest>
+----
+====
+
+The JSON payload with the signature content should have the following structure:
+
+====
+----
+{
+  "version": 2,
+  "type":    "atomic",
+  "name":    "sha256:4028782c08eae4a8c9a28bf661c0a8d1c2fc8e19dbaae2b018b21011197e1484@cddeb7006d914716e2728000746a0b23",
+  "content": "<base64 encoded signature>",
+}
+----
+====
+
+The `name` field contains the name of the image signature, which has to be unique and in
+the following format: `sha256:<digest>@<name>`. The `name` has to be 32 character long.
+
+[NOTE]
+====
+In order to attach the signature to the Docker Image the user needs to have the `system:image-signer` role.
+====
+
+==== Reading Image Signatures
+Assuming that you have already pushed signed Docker Image into {product-title} integrated Docker
+Registry, you can read the signatures using the following command:
+
+====
+----
+GET /extensions/v2/{namespace}/{name}/signatures/sha256:{digest}
+----
+====
+
+====
+----
+$ curl http://<user>:<token>@<registry-endpoint>:5000/extensions/v2/<namespace>/<name>/signatures/sha256:<digest>
+----
+====
+
+Where the `namespace` represents the {product-title} project name or Docker Registry repository name and the `name`
+refers to the name of the Image. The `digest` represents the SHA256 checksum of the Image.
```

Comment:
tried to fix them all.. in some cases I was refering to image and in some cases to API Image object... I dunno if we make distincion between them.